### PR TITLE
add ram and cpu allocation ration

### PR DESCRIPTION
**Pull request by lachie83**

### CODE REVIEW

**Comment by lachie83** [comment]
Only added to kilo at this point. Can you confirm that only the controller nodes need this change?

**Comment by pupapaik** [comment]
It should be enough for nova scheduler only. See this http://www-01.ibm.com/support/knowledgecenter/SS8MU9_2.2.0/Admin/tasks/settingresourceovercommitratios.html

We need to configure scheduler_filters too.

**Comment by lachie83** [comment]
@pupapaik I've moved everything to metadata and included defaults, removed conditionals and updated example pillar in README. LMK if this looks better.

File: nova/files/kilo/nova-controller.conf.Debian [hunk 1]
```diff
@@ -20,6 +20,17 @@ libvirt_use_virtio_for_bridges=True
 libvirt_inject_partition = -1
 vif_plugging_is_fatal = False
 vif_plugging_timeout = 0
+cpu_allocation_ratio = {{ controller.cpu_allocation_ratio }}
+ram_allocation_ratio = {{ controller.ram_allocation_ratio }}
+scheduler_default_filters = {{ controller.scheduler_default_filters }}
+
+{%- if controller.cpu_allocation_ratio is defined %}
```
Comment (pupapaik) [inline nova/files/kilo/nova-controller.conf.Debian]: There is still condition from the first commit. Please remove them.

**Comment by lachie83** [comment]
Sorry about that. Updated.

**Comment by pupapaik** [comment]
Looks good to me. Thanks

**Pull request merged**